core/controllers/profile_test.py

```diff
@@ -464,10 +464,11 @@ def test_user_bio_exceeds_limit(self) -> None:
                 'update_type': 'user_bio',
                 'data': 'I am not within 2000 char limit' * 200
             },
-            csrf_token=csrf_token, expected_status_int=400)
-        self.assertEqual(user_bio_response['status_code'], 400)
+            csrf_token=csrf_token, expected_status_int=500)
```

Comment:
Why did this change? This is a client-side "invalid request" error so it should be 400, right?

In general, error status codes have well-defined meaning, don't arbitrarily change them. 500 is for server-side errors (caused by the code on the server-side being faulty).

Reply:
Done